package.json

```diff
@@ -66,8 +72,21 @@
     "build": "webpack -p --config webpack.config.prod.js",
     "watch": "webpack --config webpack.config.dev.js --watch --progress --colors",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "precommit": "lint-staged"
   },
+ "lint-staged": {
+   "linters":{
+       "*.{js,jsx,css}": [
```

Comment:
Does prettier format css as well?

Reply:
Do we have any `jsx` component?

Reply:
checked prettier, it supports `css` formatting with a plugin. removing it for now. Maybe we can use [stylelint](https://stylelint.io/) or [postcss](https://postcss.org/)